WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardService.swift

```diff
@@ -92,4 +100,20 @@ private extension BlogDashboardService {
 
         return (sections, items)
     }
+
+    func createQuickActionsItems() -> [DashboardCardModel] {
+        let stats = createQuickActionsCardModel(title: "Stats", icon: .gridicon(.statsAlt))
+        let posts = createQuickActionsCardModel(title: "Posts", icon: .gridicon(.posts))
+        let media = createQuickActionsCardModel(title: "Media", icon: .gridicon(.image))
+        let pages = createQuickActionsCardModel(title: "Pages", icon: .gridicon(.pages))
+        return [stats, posts, media, pages]
+    }
+
+    func createQuickActionsCardModel(title: String, icon: UIImage) -> DashboardCardModel {
```

Comment:
Same for this method. This is a local card so it doesn't need a `viewModel`.

I still wonder how we'll manage `Hashable` items for local cards but at this moment we don't need to create that.

Reply:
Addressed by the redesign -- now I'm creating the buttons within the cell itself. 2752db7